docs/use-bubble-sort.mdx

```diff
@@ -0,0 +1,50 @@
+---
+name: Sorting
+route: /sorting
+---
+
+import { Playground, Props } from "docz"
+import { useBubbleSort } from "../src"
+
+# useBubbleSort
+
+```js
+import { useBubbleSort } from "use-data-structures"
+
+function App() {
+  const unsortedArray = [64, 25, 12, 22, 11];
+  const sortedArray = useBubbleSort(unsortedArray);
+  // you can pass a custom predicate as the second argument
+  // to change how the array will be sorted
+  // (e.g. (a, b) => a > b) to sort the list in descending order
+
+  return (
+    <div>
+      <h2>Unsorted array</h2>
+      <p>{ unsortedArray.join(",").toString() }</p>
```

Comment:
Add a space on the join argument (like so `",  "`) so the list gets prettier

Reply:
Done